docker_files_test.go

```diff
@@ -83,17 +81,9 @@ func TestCopyFileToRunningContainer(t *testing.T) {
 	require.NoError(t, err)
 
 	// Give some time to the wait script to catch the hello script being created
-	time.Sleep(200 * time.Millisecond)
-
-	logs, err := container.Logs(ctx)
-	require.NoError(t, err)
-
-	logsString := new(strings.Builder)
-	_, err = io.Copy(logsString, logs)
+	err = wait.ForLog("done").WithStartupTimeout(200*time.Millisecond).WaitUntilReady(ctx, container)
```

Comment:
😮 I didn't know you could use wait strategies like this!

Reply:
I think this is cool to use, and more deterministic than a fixed sleep, that could be different/slower/faster at CI time